packages/plugins/Solana/src/UI/SNSAdaptor/index.tsx

```diff
@@ -8,7 +8,7 @@ import { Web3UI } from '../Web3UI'
 const sns: Plugin.SNSAdaptor.Definition = {
     ...base,
     async init(signal, context) {
-        setupStorage(context.createKVStorage('memory', StorageDefaultValue))
+        setupStorage(context.createKVStorage('persistent', StorageDefaultValue))
```

Comment:
May I ask the reason for this change?

Reply:
To restore data in next browser session.

Reply:
Better avoid doing this.

Reply:
If you're gonna store this, please make sure you only store the needed information and leave the rest in the memory.